src/Helpers/HelperTrait.php

```diff
@@ -2,8 +2,27 @@
 
 namespace GingerPluginSdk\Helpers;
 
+use GingerPluginSdk\Collections\AbstractCollection;
+
 trait HelperTrait
 {
+    public static function isSameType($argument1, $argument2): bool
```

Comment:
I would suggest to use first_argument, second_argument (or something else like first_object, second_object) instead of using a number, it seems weird to see a number in var name, but it's up to you!

Reply:
It's was a quick solution, will think how to name this in a best way